### ISSUE TITLE
Why does this module still require react v0.14 ?

### ISSUE THREAD

**Issue by srcspider** [tomchentw/react-google-maps#241]
I use npm2, due to various things missing in npm3 as well as performance issues with npm3, and this `peerDependencies` constraint prevents this module from being installed allongside react v15. 

```json
"peerDependencies": {
  "react": "^0.14.0",
  "react-dom": "^0.14.0"
}
```

Can this requirement just be removed? Please. :)

**Comment by Tux1** [comment]
the module doesn't work well with React 15
And you ?

**Comment by tomchentw** [comment]
Closed by #237. Published [v4.10.0](https://github.com/tomchentw/react-google-maps/releases/tag/v4.10.0)
